raiden/ui/cli.py

```diff
@@ -745,6 +754,20 @@ def run(ctx, **kwargs):
     print('Welcome to Raiden, version {}!'.format(get_system_spec()['raiden']))
     from raiden.ui.console import Console
     from raiden.api.python import RaidenAPI
+    from raiden.utils import address_checksum_and_decode
+    import toml
+
+    if kwargs['config_file']:
+        d = toml.load(kwargs['config_file'])
+        for key, value in d.items():
```

Comment:
calling this `option_name` and `option_value` would make this more readable. Also please give a better name to `d`.

Reply:
i changed the name, d => config_file